src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.DragDrop.cs

```diff
@@ -28,6 +28,8 @@ partial class ListViewBase
 		public event DragItemsStartingEventHandler DragItemsStarting;
 		public event TypedEventHandler<ListViewBase, DragItemsCompletedEventArgs> DragItemsCompleted;
 
+		protected bool _isProcessingReorder;
```

Comment:
Does it need to be protected?

```suggestion
		private protected bool _isProcessingReorder;
```

Reply:
Your right ... I moved it without changing the modifier, will fix in another PR